### PR TITLE
Add documentation for using Google Analytics with Chrome Web Store

### PR DESCRIPTION
**Pull request by oliverdunk**
Adds documentation for using Google Analytics with Chrome Web Store.

Fixes https://github.com/GoogleChromeLabs/Extension-Docs/issues/95

### CODE REVIEW

**Comment by AmySteam** [comment]
@oliverdunk Quick ask: Can you add a link to this doc on this section? Thanks! 💯 

https://developer.chrome.com/docs/webstore/manage/#google-analytics

File: site/en/docs/webstore/google-analytics/index.md
Comment (sebastianbenz) [inline site/en/docs/webstore/google-analytics/index.md:10]:
```suggestion
your Chrome Web Store listing in addition to the view offered in the Developer Dashboard.
```

**Review by sebastianbenz — APPROVED** [review]
Great guide @oliverdunk! Do you think it would make sense to mention that it's possible to access and share WebStore analytics data via Google Datastudio?

**Comment by oliverdunk** [comment]
> Great guide @oliverdunk! Do you think it would make sense to mention that it's possible to access and share WebStore analytics data via Google Datastudio?

Good call! I added a brief note about this, let me know your thoughts.

**Comment by sebastianbenz** [comment]
Perfect - thanks!

File: site/en/docs/webstore/google-analytics/index.md
Comment (jpmedley) [inline site/en/docs/webstore/google-analytics/index.md:86]:
```suggestion
Go to **Admin**, then **Conversions** and choose **New Conversion Event**. Enter "install" and click **Save**. The
```

Comment (jpmedley) [reply site/en/docs/webstore/google-analytics/index.md:86]: This is per the corporate style guide and long-standing industry convention.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/webstore/google-analytics/index.md
Comment (oliverdunk) [reply site/en/docs/webstore/google-analytics/index.md:30]: Do you mind saying more about this? There aren't any limits that you can change, so I don't think that wording quite works.

File: site/en/docs/webstore/google-analytics/index.md
Comment (oliverdunk) [reply site/en/docs/webstore/google-analytics/index.md:49]: I'm not aware of anything else.

File: site/en/docs/webstore/google-analytics/index.md
Comment (oliverdunk) [reply site/en/docs/webstore/google-analytics/index.md:51]: What would the difference be?

File: site/en/docs/webstore/google-analytics/index.md
Comment (oliverdunk) [reply site/en/docs/webstore/google-analytics/index.md:52]: I'll tweak this to make that clearer.

File: site/en/docs/webstore/google-analytics/index.md
Comment (oliverdunk) [reply site/en/docs/webstore/google-analytics/index.md]: I've tried rewording this, let me know if that looks good :)

**Review by jpmedley — APPROVED** [review]

**Comment by oliverdunk** [comment]
Just waiting for an LGTM from David and then I'll merge.

**Review by dasli — APPROVED** [review]
LGTM

**Comment by dasli** [comment]
One note: This document will likely need to be updated soon as the track_install goes away. It's not recommended for developers to use track_install: https://groups.google.com/a/chromium.org/g/chromium-extensions/c/8mLf39E7i2c/m/4dWNpdVrBQAJ

Do we want to mention this at all?

File: site/en/docs/webstore/google-analytics/index.md
Comment (jpmedley) [inline site/en/docs/webstore/google-analytics/index.md:14]: Should this have some background on why developers need to do this?

You can get some information from the [MSA we sent out](https://docs.google.com/document/d/1RHxo1dMGGvFD7x-Ml77cu3CB-LE03EIBCK_SwCA8rU4/edit). Please don't write it like an advertisement for the new shiny (as we did in the MSA). Language like that will just sound weird a year from now.

Comment (oliverdunk) [reply site/en/docs/webstore/google-analytics/index.md:14]: I don't think so (going to resolve for now), but happy to chat more and update if needed. I specifically didn't include anything like that to avoid making this specific to the current transition. This page hopefully just answers the question "How do I see who's finding my extension" without making any reference to the history.

**Pull request merged**
